back-end/hub-codegen/src/test/java/io/apicurio/hub/api/codegen/OpenApi2QuarkusTest.java

```diff
@@ -98,6 +98,9 @@ private void doFullTest(String apiDef, boolean updateOnly, String expectedFilesP
                         System.out.println(actual);
                         System.out.println("-----");
                     }
+                    if(name.contains("OrgsResource")){
+                        System.out.println("aa");
+                    }
```

Comment:
I'm guessing this was included by mistake.  :)

Reply:
Thanks ! I removed and pushed it :D